src/utils/dayjs.ts

```diff
@@ -0,0 +1,16 @@
+import dayjs from "dayjs";
+import pluralize from "pluralize";
+
+export const formatTimePast = (date: Date) => {
```

Comment:
I think you're looking for https://day.js.org/docs/en/durations/humanize :)

Reply:
Yes, that helps with formatting, thanks!